SignalServiceKit/src/Messages/OWSOutgoingNullMessage.m

```diff
@@ -35,7 +36,7 @@ - (instancetype)initWithContactThread:(TSContactThread *)contactThread
 
 #pragma mark - override TSOutgoingMessage
 
-- (NSData *)buildPlainTextData
+- (NSData *)buildPlainTextData:(SignalRecipient *)recipient
 {
     OWSSignalServiceProtosContentBuilder *contentBuilder = [OWSSignalServiceProtosContentBuilder new];
```

Comment:
Hopefully we'll add content types pretty rarely.  I'm not sure it's worth the complexity, since we could just as easily forget to use the wrapped content builder.